tests/ui/treat-err-as-bug/delay_span_bug.stderr

```diff
@@ -4,7 +4,7 @@ error: internal compiler error: delayed span bug triggered by #[rustc_error(dela
 LL | fn main() {}
    | ^^^^^^^^^
 
-error: internal compiler error: unexpected panic
+error: the compiler unexpectedly panicked. this is a bug.
```

Comment:
Actually, now that I think of it, @matthiaskrgr it should be equally valid to grep for something like "this is a bug" for delayed bug cases like this, rather than "unexpected panic". Is that ok?

Reply:
`rg`'ed in my local `cargo cache` (3K crates, 81 git checkouts)

* `internal compiler error: ` 0
* `unexpected panic` 14
* `this is a bug` 109
* `the compiler unexpectedly panicked`: 0

so  "internal compiler error" and "the compiler unexpectedly panicked" gives us the most information because we can almost always be sure, that this is NOT a false positive from for example a `format!("internal compiler error")` that needed an extra `&` and sneaked into a diagnostic that way...
"this is a bug" however gives us the least amount of information because a bunch of other crates actually use this in their code.

Reply:
well "unexpectedly panicked" should also work as a good grep string then?